test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @summary Stress test virtual threads with a variation of the Skynet 1M benchmark
  * @requires vm.continuations
- * @run main/othervm/timeout=300 -Xmx1g Skynet
+ * @run main/othervm/timeout=350 -Xmx1g Skynet
```

Comment:
I assume this should be dropped from this PR as it is nothing to do with implement the AIX version of PollerProvider.

Reply:
Agreed. It feels funny to push code I know will cause a test failure. But, I agree that it simplifies the git history, and aligns with the stated purpose of this PR. Thanks for the suggestion. I have made this change.